### PR TITLE
Audio Response: update to match ui of other response components

### PR DESCRIPTION
**Pull request by rwaldron**
In Chrome: 

image

In Mobile Safari: 

image

In the editor: 

image
image

image

### CODE REVIEW

File: client/components/Slide/Components/AudioResponse/Display.jsx [hunk 1]
```diff
@@ -75,40 +75,46 @@ class Display extends Component {
...
+                            Done
+                        </Button>
+                    )}
+                </Segment>
+
+                {this.state.blobURL && (
                     <audio src={this.state.blobURL} controls="controls" />
-                </React.Fragment>
-            )) || <p>Please switch to Chrome or Firefox to record audio.</p>
+                )}
```
Comment (evmiguel) [inline client/components/Slide/Components/AudioResponse/Display.jsx]: YES. This is how the previous version of TM does it

File: client/components/Slide/Components/AudioResponse/Display.jsx [hunk 2]
```diff
@@ -75,40 +75,46 @@ class Display extends Component {
...
+            </React.Fragment>
+        ) : (
```
Comment (evmiguel) [inline client/components/Slide/Components/AudioResponse/Display.jsx]: YES. This is how the previous version of TM does it

File: client/components/Slide/Components/AudioResponse/Editor.jsx [hunk 3]
```diff
@@ -26,18 +26,23 @@ class AudioResponseEditor extends Component {
...
+                            <Input
+                                label="Audio Prompt:"
+                                name="prompt"
+                                value={prompt}
+                                onChange={onTextAreaChange}
+                            />
+                        }
+                    />
+
+                    <Message content="Note: This component will fallback to a text input prompt when Audio recording is not supported." />
```
Comment (evmiguel) [inline client/components/Slide/Components/AudioResponse/Editor.jsx]: Nice

**Comment by rwaldron** [comment]
Thank you both!!

**Pull request merged**